### PR TITLE
Minor corrections and stuff

### PR DESCRIPTION
**Pull request by raimohanska**
Briefly checked the comparison doc. This is good stuff. Big thums up!

### CODE REVIEW

**Comment by pozadi** [comment]
Thanks! Was going to ask you for corrections :) 
Still a bit confused though, will add comments to the diff.

File: bacon-vs-kefir-api.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 | ----- | ----- | -------- |
 | `$.fn.asEventStream(eventName, [selector], [eventTransformer])` | `$.fn.asKefirStream(eventName, [selector], [eventTransformer])` | In Kefir this method available not in core but in separate lib [kefir-jquery](https://github.com/pozadi/kefir-jquery) |
 | `Bacon.fromPromise(promise, [abort])` | `Kefir.fromPromise(promise)` | No `abort` option in Kefir, and the result is a Property unlike Stream in Bacon |
-| `Bacon.fromEventTarget(target, eventName, [eventTransformer])` | `Kefir.fromEvent(target, eventName, [eventTransformer])` |  |
+| `Bacon.fromEvent(target, eventName, [eventTransformer])` | `Kefir.fromEvent(target, eventName, [eventTransformer])` |  |
```
Comment (pozadi) [inline bacon-vs-kefir-api.md]: This was from Bacon docs :)

File: bacon-vs-kefir-api.md [hunk 2]
```diff
@@ -155,11 +155,11 @@
 | ----- | ----- | -------- |
 | `stream.map(property)` | Use `.sampledBy()` |  |
 | `obs.filter(property)` | `obs.filterBy(obs)` |  |
-| `obs.takeWhile(property)` | `obs.takeWhileBy(obs)` | Only property supported in Bacon? |
+| `obs.takeWhile(property)` | `obs.takeWhileBy(obs)` | Bacon supports property or predicate, Kefir supports any observable |
```
Comment (pozadi) [inline bacon-vs-kefir-api.md]: There is versions of these methods with predicate in the "Modify an observable" section. I think we'd better add something like "see also `takeWhile(fn)` above", to not confuse anybody that Kefir doesn't have version with predicate.

File: bacon-vs-kefir-api.md [hunk 3]
```diff
@@ -155,11 +155,11 @@
...
 | `obs.combine(obs2, fn)` | `obs.combine(obs2, [fn])` | In Kefir `fn` optional |
-| `stream.skipWhile(property)` | `obs.skipWhileBy(otherObs)` | Only property supported in Bacon? |
-| `stream.skipUntil(stream2)` | `obs.skipUntilBy(otherObs)` | Only stream supported in Bacon? |
-| `obs.takeUntil(stream) ` | `obs.takeUntilBy(otherObs)` | Only stream supported in Bacon? |
+| `stream.skipWhile(property)` | `obs.skipWhileBy(otherObs)` | Bacon supports property or predicate, Kefir supports any observable |
+| `stream.skipUntil(stream2)` | `obs.skipUntilBy(otherObs)` |  |
+| `obs.takeUntil(stream) ` | `obs.takeUntilBy(otherObs)` |  |
```
Comment (pozadi) [inline bacon-vs-kefir-api.md]: Does that mean that `.skipUntil` and `.takeUntil` supports any observable in both positions? We should fix the signatures then too. Of if signatures correct, there should be a note.

Comment (raimohanska) [inline bacon-vs-kefir-api.md]: Oh indeed in Bacon it is `stream.take/skipWhile(obs)`... So Property supported in second position only.

File: bacon-vs-kefir-api.md (hunk 2 above)
Comment (raimohanska) [inline bacon-vs-kefir-api.md]: True that!

File: bacon-vs-kefir-api.md (hunk 1 above)
Comment (raimohanska) [inline bacon-vs-kefir-api.md]: Well yeah. That's the old one. Who reads docs anyway? :) I'll update...

**Comment by pozadi** [comment]
Ok, merging this, will make changes based on comments after merge. Thanks!

**Pull request merged**